package.json

```diff
@@ -61,6 +66,8 @@
     "eslint": "eslint ./{src,electron}/. --ext .js",
     "prettier": "prettier --write '{electron,src}/**/*.{js,jsx,json,css,scss,md}'",
     "test": "react-scripts test",
+    "test-react": "react-scripts test",
+    "test-electron": "jest --detectOpenHandles electron/**/*.js --watch --verbose",
```

Comment:
Oh, doesn't react-scripts test run also jest tests? I prefer having one command which runs all the tests.

Reply:
Aah, now I understand, it's that --watch! And that `yarn test` is run in Github Actions with env `CI=true` which excludes watch.